package.json

```diff
@@ -112,6 +112,7 @@
     "cozy-scanner": "0.2.0",
     "cozy-scripts": "1.13.2",
     "cozy-sharing": "^1.8.0",
+    "cozy-stack-client": "11.2.0",
```

Comment:
This is not a dev dep, right? We didn't add it only for tests?

The main problem I see with this is that we now need to upgrade both packages manually :(

Reply:
Nope. This is not for the test :/